### PR TITLE
KFSPTS-8397: Update balance type and total on SAE-to-Collector GL files

### PR DESCRIPTION
**Pull request by cah292**
As per the related user story, this PR changes the SAE-to-Collector process so that the Collector file rows now have the "AC" ("Actual") balance type, and so that the trailer row now contains the total of just the debit lines.

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java [hunk 1]
```diff
@@ -201,22 +208,32 @@ protected void groupLines(List<ConcurStandardAccountingExtractDetailLine> saeLin
     }
 
     protected void updateCollectorBatchWithOriginEntries() {
+        Integer fiscalYear = Integer.valueOf(collectorBatch.getUniversityFiscalYear());
+        SystemOptions options = optionsService.getOptions(fiscalYear);
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java]: Would it make sense to rename this fiscalYearOptions for clarity?

Comment (cah292) [inline src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java]: Ok, I'll rename this to be more descriptive.

**Comment by cah292** [comment]
I've updated the code based on Jay's suggestions. This is now ready for review again.

**Comment by cah292** [comment]
retest this please

**Comment by cah292** [comment]
Thanks for reviewing @jhulslander 

@nkimble If you have any concerns about this PR that you think should require reverting it, please let me know.

**Comment by jhulslander** [comment]
I'm sorry @nkimble , I didn't notice the multiple reviewers.

**Pull request merged**